docs/IR.md

```diff
@@ -55,12 +55,13 @@ The following describes the metadata properties of a model graph:
 
 #### Names Within a Graph
 
-Names are organized into separate namespaces, and must be unique within a namespace. The namespaces include the following:
- - `Node`: names that identify specific nodes in the graph, but not necessarily any particular input or output of the node.
- - `Graph`: names that identify graphs in the protobuf.
+Names are organized into separate namespaces. The namespaces include the following:
+ - `Node`: names that identify specific nodes in the graph, but not necessarily any particular input or output of the node. Each node name must be unique.
+ - `Graph`: names that identify graphs in the protobuf. Each graph name must be unique.
  - `Attribute`: names that identify attribute names for extra attributes that are passed to operators.
- - `Operator`: names that identify particular operators.
- - `Tensor`: names that identify intermediate tensor values flowing through the computation of a graph.
+ - `Operator`: names that identify particular operators. Each operator name must be unique.
```

Comment:
operator name should be unique in its domain.

Reply:
Domain scope unique.

Reply:
Yeah, operators don't fit it as they're more part of the spec than the model